apps/galaxy/src/services/orders/useOrders.ts

```diff
@@ -0,0 +1,56 @@
+import axios from "../utils/axios";
+import { useToast } from "@cc/ui-chakra";
+import { useEffect, useState } from "react";
+import { AxiosError } from "axios";
+import { getFranchiseeId, getToken } from "@src/utils/tokenService";
+
+export type Order = {
+  id: number;
+  createdTime: string;
+  postcode: string;
+  totalAmount: number;
+  designInformation: DesignInformation;
+};
+
+export type DesignInformation = {
+  quotation: string;
+  constructionDraw: string;
+  isNeedLevelGround: boolean;
+  constructionAddress: {
+    country: string;
+    state: string;
+    city: string;
+    line1: string;
+    line2: string;
+    postalCode: string;
+  };
+};
+
+export default function useGetOrders() {
```

Comment:
Try to use const to keep the code consistency

Reply:
Oh, I forgot to push that commit.
Resolved
Thanks.